doc/2/plugins/guides/events/realtime/index.md

```diff
@@ -0,0 +1,257 @@
+
+:::info
+Pipes cannot listen to that event, only hooks can.
+:::
+
+| Arguments | Type              | Description            |
+| --------- | ----------------- | ---------------------- |
+| `roomId`  | <pre>string</pre> | Room unique identifier |
+
+
+---
+
+## core:realtime:user:subscribe:after
+
+Triggered whenever a user makes a new [subscription](/core/2/api/controllers/realtime/subscribe).
+
+<SinceBadge version="auto-version"/>
+
+| Arguments  | Type              | Description                           |
+| ---------- | ----------------- | ------------------------------------- |
+| `subscription`     | <pre>object</pre> | Contains information about the added subscription |
+
+
+### subscription
+
+The provided `subscription` object has the following properties:
+
+| Properties     | Type                 | Description                                                                                                       |
+| -------------- | -------------------- | --------------------------------------------------------------------------------------------------------- |
+| `roomId`       | <pre>string</pre>    | Room unique identifier                                                                                    |
+| `connectionId` | <pre>integer</pre>   | [ClientConnection](/core/2/protocols/api/context/clientconnection) unique identifier                      |
+| `index`        | <pre>string</pre>    | Index                                                                                                     |
+| `collection`   | <pre>string</pre>    | Collection                                                                                                |
+| `filters`      | <pre>object</pre>    | Filters in [Koncorde's normalized format](https://www.npmjs.com/package/koncorde#filter-unique-identifier)|
+
+---
+
+## core:realtime:user:unsubscribe:after
+
+Triggered whenever a user leaves a room.
+
+<SinceBadge version="auto-version"/>
+
+| Arguments        | Type              | Description                                                                                  |
+| -----------------| ----------------- | -------------------------------------------------------------------------------------------- |
+| `RequestContest` | <pre>object</pre> | [requestContext](/core/2/protocols/api/context/requestcontext/) object |
+| `room`           | <pre>object</pre> | Joined room information in Koncorde format                                                   |
+
+### room
+
+The provided `room` object has the following properties:
+
+| Properties     | Type                 | Description             |
+| -------------- | -------------------- | ----------------------- |
+| `id`           | <pre>string</pre>    | Room unique identifier  |
+| `index`        | <pre>string</pre>    | Index                   |
+| `collection`   | <pre>string</pre>    | Collection              |
+
+---
+
+## notify:dispatch
```

Comment:
According to the new naming, this event and the following shouldn't be `notify:dispatch:before` ?

Reply:
That would be a breaking change, unfortunately. 
Same goes for all our API events named `controller:[before|after]Action`, which should really be named: `api:controller:action:[before|action]`

That'll be for a next major version.